src/Microsoft.Health.Dicom.Blob/Features/Storage/BlobFileStore.cs

```diff
@@ -142,6 +142,26 @@ public class BlobFileStore : IFileStore
         }
     }
 
+    public async Task<Stream> GetStreamingFileAsync(
+        VersionedInstanceIdentifier versionedInstanceIdentifier,
+        CancellationToken cancellationToken)
+    {
+        EnsureArg.IsNotNull(versionedInstanceIdentifier, nameof(versionedInstanceIdentifier));
+
+        BlockBlobClient blobClient = GetInstanceBlockBlobClient(versionedInstanceIdentifier, _blobMigrationFormatType);
+
+        Stream stream = null;
+        var blobOpenReadOptions = new BlobOpenReadOptions(allowModifications: false);
+
+        await ExecuteAsync(async () =>
+        {
+            Response<BlobDownloadStreamingResult> result = await blobClient.DownloadStreamingAsync(range: default, conditions: null, rangeGetContentHash: false, cancellationToken);
```

Comment:
I know we had a number of conversations during the hackathon surrounding the various blob APIs we could use during export.  Has something changed, or could we always simply stream the blob to the destination without buffering?

Reply:
Not yet, we need to follow up with the Blob team to see, if the new SDK is available